browser-test/src/applicant/northstar_application_version_fast_forward.test.ts

```diff
@@ -851,15 +851,17 @@ test.describe(
             ])
           })
 
-          return await test.step('add questions to program and republish program', async () => {
+          await test.step('add questions to program', async () => {
```

Comment:
No test functionality changes. Only updated the comments because the program being disabled wasn't clear when reading the test logs